tokio/src/sync/loom.rs

```diff
@@ -0,0 +1,48 @@
+#[cfg(not(all(test, loom)))]
+mod imp {
```

Comment:
Do we want to merge this and the `mod loom` used by `tokio::executor` while we're at it?

Reply:
I think the intention is to consolidate this (and also the `mod loom` in `tokio-net`) in a follow-up PR.

Reply:
follow up PR. I'm trying to "just" collapse crates here. There is a lot of follow up cleanup to do.